lib/config.js

```diff
@@ -36,20 +36,21 @@ config.mongodb.authentication = {
   // used by MongoDB < 3.0
   // authMechanism: 'MONGODB-CR'
 };
-
+// if true role, server version, & authn checks are skipped
+// this is appears to be necessary on some mongo servers
+config.mongodb.skipChecks = false;
```

Comment:
`skipChecks` is not a good name because it isn't clear what it does. Can you suggest something more clear about its function?

Reply:
You may be able to get a better name by inverting the flag to express what happens when the flag is *on* (and make that the new default) as well.

Reply:
I changed it to `checkServerDetails` here: https://github.com/digitalbazaar/bedrock-mongodb/pull/89/commits/0db895e4351f66b66ad05b62ea3e4a0dd19db8fb

That seems better, the problem is we are skipping 3 checks: serverVersion, role check, and then the requiresAuthn check.

Reply:
This feature was removed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/89/commits/7718dca684f277d75621c137c5c9580b77870f0c